services/web/client/source/class/osparc/auth/Manager.js

```diff
@@ -148,6 +148,8 @@ qx.Class.define("osparc.auth.Manager", {
     __loginUser: function(profile) {
       osparc.auth.Data.getInstance().setEmail(profile.login);
       osparc.auth.Data.getInstance().setToken(profile.login);
+      osparc.auth.Data.getInstance().setUserId(profile.id);
+      osparc.auth.Data.getInstance().setGroupId(profile["groups"]["me"]["gid"]);
```

Comment:
MINOR: why are you not using ``profile.groups.me.gid`` here as you do with the others?